server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -19,106 +20,102 @@ import {
-        response.send(result)
+
+    symptomsLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_SYMPTOMS, response.locals)
+    .then(result => {
+        symptomsLogger.info(validDBResponseLog, Severity.LOW);
+        response.send(result);
     }).catch(error => {
-        symptomsLogger.error(`got error when approaching the graphql API: ${error}`, Severity.HIGH);
-        response.sendStatus(errorStatusCode);
+        symptomsLogger.error(invalidDBResponseLog(error), Severity.HIGH);
+        response.sendStatus(error);
     });
 });
 
 clinicalDetailsRoute.get('/backgroundDiseases', (request: Request, response: Response) => {
     const backgroundDiseasesLogger = logger.setup({
-        workflow: 'Fetching Background Diseases',
+        workflow: 'query all background diseases',
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     });
-    backgroundDiseasesLogger.info('launching DB request', Severity.LOW);
-    graphqlRequest(GET_BACKGROUND_DISEASES, response.locals).then((result: any) => {
-        backgroundDiseasesLogger.info('got respond from DB', Severity.LOW);
-        response.send(result)
+
+    backgroundDiseasesLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_BACKGROUND_DISEASES, response.locals)
+    .then(result => {
+        backgroundDiseasesLogger.info(validDBResponseLog, Severity.LOW);
+        response.send(result);
     }).catch(error => {
-        backgroundDiseasesLogger.error(`got error when approaching the graphql API: ${error}`, Severity.HIGH);
-        response.sendStatus(errorStatusCode);
+        backgroundDiseasesLogger.error(invalidDBResponseLog(error), Severity.HIGH);
+        response.sendStatus(error);
     });
 });
 
 clinicalDetailsRoute.get('/isolationSources', (request: Request, response: Response) => {
     const isolationSourcesLogger = logger.setup({
-        workflow: 'Fetching Isolation Source',
+        workflow: 'query all isolation sources',
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     })
-    isolationSourcesLogger.info('start DB request', Severity.LOW);
-    graphqlRequest(GET_ISOLATION_SOURCES, response.locals).then((result: any) => {
-        if(result?.data) {
-            isolationSourcesLogger.info('got response from DB', Severity.LOW);
-            response.send(result?.data?.allIsolationSources?.nodes)
-        } else {
-            isolationSourcesLogger.error(`couldnt query isolation sources due to ${result?.errors[0]?.message}`, Severity.HIGH);
-            response.sendStatus(errorStatusCode);
-        }
+
+    isolationSourcesLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_ISOLATION_SOURCES, response.locals)
+    .then(result => {
+        isolationSourcesLogger.info(validDBResponseLog, Severity.LOW);
+        response.send(result?.data?.allIsolationSources?.nodes || []);
```

Comment:
maybe put into const and send the const?

Reply:
Why? You don’t use this const anywhere else